static/js/main.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function(){
+  if ($('.bar').length > 0) {
+    setTimeout(function start (){
+      
+      $('.bar').each(function(i){  
+        var $bar = $(this);
+        $(this).append('<span class="count"></span>')
+        setTimeout(function(){
+          $bar.css('width', $bar.attr('data-percent'));      
+        }, i*100);
+      });
+
+    $('.count').each(function () {
+        $(this).prop('Counter',0).animate({
+            Counter: $(this).parent('.bar').attr('data-percent')
```

Comment:
What's the point of doing this?

Reply:
This code has been used to make a jquery animated counter that is there on the right side of the bar of the graph.
For reference, I've used the following link:
https://stackoverflow.com/questions/23006516/jquery-animated-number-counter-from-zero-to-value